[Parallel(n_jobs=-1)]: Using backend LokyBackend with 24 concurrent workers.
Memmapping (shape=(808, 7268320), dtype=float64) to new file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(673,), dtype=int64).
Pickling array (shape=(135,), dtype=int64).
Memmapping (shape=(808, 7268320), dtype=float64) to old file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(673,), dtype=int64).
Pickling array (shape=(135,), dtype=int64).
Memmapping (shape=(808, 7268320), dtype=float64) to old file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(673,), dtype=int64).
Pickling array (shape=(135,), dtype=int64).
Memmapping (shape=(808, 7268320), dtype=float64) to old file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(673,), dtype=int64).
Pickling array (shape=(135,), dtype=int64).
Memmapping (shape=(808, 7268320), dtype=float64) to old file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(674,), dtype=int64).
Pickling array (shape=(134,), dtype=int64).
Memmapping (shape=(808, 7268320), dtype=float64) to old file /dev/shm/joblib_memmapping_folder_13844_3186619410/13844-46942505174408-c6f0d3def6c3440d818d5e6f3ff320d2.pkl
Pickling array (shape=(808,), dtype=int64).
Pickling array (shape=(674,), dtype=int64).
Pickling array (shape=(134,), dtype=int64).
[Parallel(n_jobs=-1)]: Done   1 tasks      | elapsed:  5.7min
[Parallel(n_jobs=-1)]: Done   2 out of   6 | elapsed:  5.9min remaining: 11.8min
[Parallel(n_jobs=-1)]: Done   3 out of   6 | elapsed:  6.3min remaining:  6.3min
[Parallel(n_jobs=-1)]: Done   4 out of   6 | elapsed:  6.6min remaining:  3.3min
[Parallel(n_jobs=-1)]: Done   6 out of   6 | elapsed:  8.3min remaining:    0.0s
[Parallel(n_jobs=-1)]: Done   6 out of   6 | elapsed:  8.3min finished
SVC Mean CV Score: nan
[CV]  ................................................................
[CV] ...................................... , score=nan, total= 5.2min
/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py:552: FitFailedWarning: Estimator fit failed. The score on this train-test partition for these parameters will be set to nan. Details: 
Traceback (most recent call last):
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py", line 531, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 330, in fit
    Xt = self._fit(X, y, **fit_params_steps)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 296, in _fit
    **fit_params_steps[name])
  File "/quarantine/PYTHON/anaconda3/envs/py36/lib/python3.6/site-packages/joblib/memory.py", line 355, in __call__
    return self.func(*args, **kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 740, in _fit_transform_one
    res = transformer.fit_transform(X, y, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/base.py", line 693, in fit_transform
    return self.fit(X, y, **fit_params).transform(X)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 353, in fit
    score_func_ret = self.score_func(X, y)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 151, in f_classif
    return f_oneway(*args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 95, in f_oneway
    args = [as_float_array(a) for a in args]
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 95, in <listcomp>
    args = [as_float_array(a) for a in args]
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/validation.py", line 164, in as_float_array
    return X.copy('F' if X.flags['F_CONTIGUOUS'] else 'C') if copy else X
MemoryError: Unable to allocate array with shape (334, 7268320) and data type float64

  FitFailedWarning)
[CV]  ................................................................
[CV] ...................................... , score=nan, total= 5.4min
/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py:552: FitFailedWarning: Estimator fit failed. The score on this train-test partition for these parameters will be set to nan. Details: 
Traceback (most recent call last):
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py", line 531, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 330, in fit
    Xt = self._fit(X, y, **fit_params_steps)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 296, in _fit
    **fit_params_steps[name])
  File "/quarantine/PYTHON/anaconda3/envs/py36/lib/python3.6/site-packages/joblib/memory.py", line 355, in __call__
    return self.func(*args, **kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 740, in _fit_transform_one
    res = transformer.fit_transform(X, y, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/base.py", line 693, in fit_transform
    return self.fit(X, y, **fit_params).transform(X)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 353, in fit
    score_func_ret = self.score_func(X, y)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 151, in f_classif
    return f_oneway(*args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 98, in f_oneway
    ss_alldata = sum(safe_sqr(a).sum(axis=0) for a in args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 98, in <genexpr>
    ss_alldata = sum(safe_sqr(a).sum(axis=0) for a in args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/__init__.py", line 712, in safe_sqr
    X = X ** 2
MemoryError: Unable to allocate array with shape (335, 7268320) and data type float64

  FitFailedWarning)
[CV]  ................................................................
[CV] ...................................... , score=nan, total= 5.8min
/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py:552: FitFailedWarning: Estimator fit failed. The score on this train-test partition for these parameters will be set to nan. Details: 
Traceback (most recent call last):
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py", line 531, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 330, in fit
    Xt = self._fit(X, y, **fit_params_steps)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 296, in _fit
    **fit_params_steps[name])
  File "/quarantine/PYTHON/anaconda3/envs/py36/lib/python3.6/site-packages/joblib/memory.py", line 355, in __call__
    return self.func(*args, **kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 740, in _fit_transform_one
    res = transformer.fit_transform(X, y, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/base.py", line 693, in fit_transform
    return self.fit(X, y, **fit_params).transform(X)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 353, in fit
    score_func_ret = self.score_func(X, y)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 150, in f_classif
    args = [X[safe_mask(X, y == k)] for k in np.unique(y)]
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 150, in <listcomp>
    args = [X[safe_mask(X, y == k)] for k in np.unique(y)]
MemoryError: Unable to allocate array with shape (340, 7268320) and data type float64

  FitFailedWarning)
[CV]  ................................................................
[CV] ...................................... , score=nan, total= 6.0min
/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py:552: FitFailedWarning: Estimator fit failed. The score on this train-test partition for these parameters will be set to nan. Details: 
Traceback (most recent call last):
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/model_selection/_validation.py", line 531, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 330, in fit
    Xt = self._fit(X, y, **fit_params_steps)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 296, in _fit
    **fit_params_steps[name])
  File "/quarantine/PYTHON/anaconda3/envs/py36/lib/python3.6/site-packages/joblib/memory.py", line 355, in __call__
    return self.func(*args, **kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/pipeline.py", line 740, in _fit_transform_one
    res = transformer.fit_transform(X, y, **fit_params)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/base.py", line 693, in fit_transform
    return self.fit(X, y, **fit_params).transform(X)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 353, in fit
    score_func_ret = self.score_func(X, y)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 151, in f_classif
    return f_oneway(*args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 98, in f_oneway
    ss_alldata = sum(safe_sqr(a).sum(axis=0) for a in args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/feature_selection/_univariate_selection.py", line 98, in <genexpr>
    ss_alldata = sum(safe_sqr(a).sum(axis=0) for a in args)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/validation.py", line 72, in inner_f
    return f(**kwargs)
  File "/nethome/kcni/posborne/.local/lib/python3.6/site-packages/sklearn/utils/__init__.py", line 712, in safe_sqr
    X = X ** 2
MemoryError: Unable to allocate array with shape (334, 7268320) and data type float64

  FitFailedWarning)
[CV]  ................................................................
[CV] .................................... , score=0.563, total= 7.5min
[CV]  ................................................................
[CV] .................................... , score=0.556, total= 7.8min
